x-pack/filebeat/docs/inputs/input-gcs.asciidoc

```diff
@@ -341,6 +347,8 @@ filebeat.inputs:
     timestamp_epoch: 1630444800
 ----
 
+NOTE: The GCS api's don't provide a direct way to filter files based on the timestamp, so the input will download all the files and then filter them based on the timestamp. This can cause a bottleneck in processing if the number of files are very high. It is recommended to use this attribute only when the number of files are limited or ample resources are available. This option scales vertically and not horizontally.
```

Comment:
```suggestion
NOTE: The GCS APIs don't provide a direct way to filter files based on the timestamp, so the input will download all the files and then filter them based on the timestamp. This can cause a bottleneck in processing if the number of files are very high. It is recommended to use this attribute only when the number of files are limited or ample resources are available. This option scales vertically and not horizontally.
```